packages/core/package.json

```
@@ -0,0 +1,78 @@
+{
+  "name": "@wpengine/headless",
```

Lerna is failing due to two packages (`packages/core/package.json`, `packages/headless/package.json`) with the same name.

Maybe for now we could go with something like this until we finalize a name?

```suggestion
  "name": "@wpengine/headless-core",
```